nixos/lib/test-driver/run_mutmut.sh

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -I nixpkgs=../../.. -p black python3Packages.mutmut python3Packages.coverage -i bash
```

Comment:
we could wrap this script using wrapProgram, adding all the runtime-dependencies to the PATH, moreover passing the to be mutated path using an argument, and use wrapPrograms -set to set it to the path.